app/views/api/__init__.py

```diff
@@ -156,7 +156,7 @@ def put(self, old, new):
         return old
 
     def patch(self, old, new):
-        for field in self.load_fields.keys():
+        for field in self.fields.keys():
```

Comment:
This change has no effect as to every patch method only load_fields can be passed.